tests/unit/mock/config/compliance/feature_compliance/ios_dual_banner_received.py

```diff
@@ -1,10 +1,10 @@
 data = {
     "compliant": False,
-    "missing": "banner exec ^C\n=========\nintended config exec banner\n-========^C\n",
-    "extra": "banner exec ^C\n=========\nactual config exec banner\n-========^C\n",
+    "missing": "banner exec ^C\n=========\nintended config exec banner\n-========^C",
+    "extra": "banner exec ^C\n=========\nactual config exec banner\n-========^C",
     "cannot_parse": True,
     "unordered_compliant": False,
     "ordered_compliant": False,
-    "actual": "hostname dual-banner\n!\nbanner exec ^C\n=========\nactual config exec banner\n-========\n^C\nbanner motd ^C\n======\nactual config motd banner\n======\n   || ($hostname) ||\n^C\n!\n",
```

Comment:
I think we need an additional test, if you only test for `banner` (and not `banner exec` does this capture both and work correctly?

Reply:
Good point, I'll add one that checks `banner` and another that checks `banner motd`

Reply:
@itdependsnetworks I added `banner`, `banner exec`, `banner motd` tests.